File: Java/figureGeometric/Triangle.java
public class Triangle extends FigureGeometric{
    private final double side;
    private double calculate;
 
    public Triangle(double side){
        super("Triangle", "Triangulo");
        this.side=side;
    }

    public double getSide(){
        return this.side;
    }
    @Override
    public double calculateArea(){
        this.calculate=(this.getSide() * this.getSide()) / 2;
        return this.calculate;
    }
}

File: PHP/areas/triangle.php
<?php
    class Triangle extends FigureGeometric {
        private float $side;
        private float $calculate;

        public function __construct(float $side){
            parent::__construct("Triangle", "Triangulo");
            $this->side = $side;
        }
         
        public function getSide(): float{
            return $this->side;
        }

        public function calculate(): float {
            $this->calculate = ($this->getSide() * $this->getSide()) / 2;
            return $this->calculate;
        }
    }
?>